spec/views/users/users_index_spec.rb

```diff
@@ -40,5 +40,10 @@
       expect(page).to have_content(user_post.comments.count)
       expect(page).to have_content(user_post.likes.count)
     end
+
```

Comment:
- [ ] This is a test for the `Posts` index page as the previous reviewer mentioned kindly consider moving these tests to the `posts_index_spec.rb` file instead